src/platforms/hosted/stlinkv2.c

```diff
@@ -849,7 +849,7 @@ static void stlink_v2_set_frequency(const uint32_t freq)
 		stlink_v2_divisor = 1U << ulog2(divisor);
 		stlink_v2_divisor /= STLINK_V2_JTAG_MUL_FACTOR;
 	} else {
-		/* Adjust the clock frequency request to result in the corrector dividor */
+		/* Adjust the clock frequency request to result in the corrector divisor */
```

Comment:
This one's complicated.. the original intent of the phrasing here is that this code calculates a divider (this is the typo here) using a divisor - in this instance we did mean divider.